stories/topics/proc-aws-deploy-existing-vpc.adoc

```diff
@@ -12,26 +12,26 @@ Complete the procedure in that section before continuing with this section.
 The following procedure uses an existing VPC network to deploy an application.
 
 .Procedure
-. For Step 1, *Create stack*, click btn:[Next].
-. For Step 2, *Specify stack details*
+. For Step 1, on the *Create stack* page, click btn:[Next].
+. For Step 2, on the *Specify stack details* page
 .. In the *Stack name* field, enter a unique stack name.
 .. In the *EC2 KeyPair* field, select your previously created EC2 keypair.
 .. In the *Select VPC* field, select *Existing*.
 .. Ignore all the parameters for *New network configuration*.
-.. In the *For Existing network configuration* field, enter the
-** *Existing VPC ID* 
-** *Existing VPC CIDR Range* 
-** *Existing Private Subnet 1 ID*
-** *Existing Private Subnet 2 ID*
+.. In the *For Existing network configuration* field, fill the following fields
```

Comment:
I don't think a field can have other fields.

In *For Existing Network configuration * enter the following information:
 ** In the * Which VPC ID should this be deployed to? * field,  enter the VpcId of your existing Virtual Private Cloud (VPC), for example, vpc-0123456789abcdef.

And other entries similar.

Reply:
Great note here. This is a section of fields, not a field in itself. Ill update to reflect this better.